### ISSUE TITLE
Intro - What is SourceCred ?

### ISSUE THREAD

**Issue by hammadj** [sourcecred/docs#53]
super basic overview introducing basic concepts and acting as a launching point to go down the rabbit hole.

**Comment by benoxmo** [comment]
instead of "For Postcapitalist" maybe "web 3" communities fits better

**Comment by benoxmo** [comment]
"Contributions in the graph earn cred if they are **connected** to other important contributions" 
> ... are **used by** other important...

**Comment by Beanow** [comment]
> instead of "For Postcapitalist communities" > "For web 3 communities" fits better

@benoxmo Interesting thought there! Because from a technology point of view, we're currently not that web 3 ready yet. GitHub plugin, Discourse plugin, Discord plugin. Other than moving tokens on-chain at some point, we're pretty web 2 in our tech stack.

So I wonder if you're thinking about web 3 as a movement rather than a tech stack, which SourceCred belongs to?

**Comment by benoxmo** [comment]
yes it's more about the movement and enhancing communities to make thinks that has been unseen so far.

We're still web 2 now but : 
1. we're here because there is some ledger/blockchain at use at some point (cred/grain management, DAOs) 
2. value flowing for communities and in a modular way  is also quite unseen

**Comment by decentralion** [comment]
I definitely think of SourceCred as a part of the web3 movement. It so happens that it depends on a lot of web2 component--but that's true of web3 in general IMO (see usage of twitter, reddit, github, etc).
